src/components/MoveButton.vue

```diff
@@ -205,10 +205,10 @@ export default class MoveButton extends Vue {
       if (times && typeof times === "number") {
         // the \b is necessary for things like '1t-a3', so the 3 is not caught
         command = command.replace(/\b[0-9]+/g, x => ('' + (parseInt(x) * times)));
-        // this is used for bids. 
-        command = command.replace(/0+/g, x => ('' + times));
       }
 
+      command = command.replace(/\b\amount\b/g, '' + (times ?? 0));
```

Comment:
`\a` in the regexp is bad.

Try `/\b[$]times\b/` or just remove the \a. But I prefer a special symbol (the `$` prefix) just in case.

Reply:
Maybe just do `.replace('$times', ...)`, no regexp, if we can't escape the dolalr sign

Reply:
Actually it works without the `\b` at the beginning:

`/\$times\b/g`